lib/puppet/parser/functions/docker_run_flags.rb

```diff
@@ -8,7 +8,7 @@ module Puppet::Parser::Functions
     subject = args[0]
 
     escape_function = if self['facts'] && self['facts']['os']['family'] == 'windows'
-                        'powershell_escape'
+                        'stdlib::powershell_escape'
                       else
                         'shell_escape'
```

Comment:
```suggestion
                        'stdlib::shell_escape'
```

Reply:
thats odd, I could sweat I searched for all instances of shell_escape previously

This one must have skipped my radar. Nice spot